tool.go

```diff
@@ -310,7 +312,7 @@ func main() {
 					if err != nil {
 						return err
 					}
-					if pkg, err = gbuild.Import(pkgPath, 0, "", nil); err != nil {
+					if pkg, err = gbuild.Import(pkgPath, 0, "", options.BuildTags); err != nil {
 						return err
 					}
 				}
```

Comment:
The very next line is an `if pkg == nil { ... gbuild.ImportDir(currentDirectory, 0) ... }` block.

If that path is taken, then build tags would be ignored, right?

To fix that, you'd have to modify https://github.com/gopherjs/gopherjs/blob/45518c130e5bd1525f20110830a4986365a153de/build/build.go#L124 somewhat significantly, it looks like. It's fine by me to leave that out of this PR.